core/versioning/txVersionChecker.go

```diff
@@ -0,0 +1,45 @@
+package versioning
+
+import (
+	"github.com/ElrondNetwork/elrond-go/core"
+	"github.com/ElrondNetwork/elrond-go/data/transaction"
+)
+
+const (
+	maskSignedWithHash = uint32(16777216)
```

Comment:
add a comment so you can tell what this number represents?

Reply:
done.